examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -2822,4 +2830,4 @@
       "networkId": 1
     }
   ]
-}
+}
```

Comment:
nit: can we keep newlines at EOF to eliminate this kind of noise in PRs?

Reply:
Does restyler-whitespace fix this, so we could just run that on .zap files?